src/utils/fishData.js

```diff
@@ -915,3 +915,6 @@ export const initFishData = () => {
 };
 
 export const fishData = fishComponents;
+export {
```

Comment:
You can just put `export` in front of the definition of `MouthExpression` on line 107.

Reply:
👍